bin/ch_misc.c

```diff
@@ -406,6 +406,19 @@ void log_ids(const char *func, int line)
    }
 }
 
+void logging_print(bool fail) {
+   TRACE("trace");
+   DEBUG("debug");
+   VERBOSE("verbose");
+   INFO("info");
+   WARNING("warning");
+   if (fail) {
+      FATAL("the program failed inexplicably");
```

Comment:
As much as I like this joke, I wonder if it will become confusing over time and the message should be more clear about what his happening.

Also you don't need braces on this `if`.